libs/domain/search/facet-navigation/facet-navigation.component.ts

```diff
@@ -22,7 +22,7 @@ import { searchFacetNavigationStyles } from './facet-navigation.styles';
   expandedItemsCount: 5,
   valueRenderLimit: 5,
   minForSearch: 13,
-  bury: [{ facets: ['rating', 'price'] }],
```

Comment:
Why we need this change?

Reply:
to show the rating component. It was hidden because there were no normalizer and component. Now it is implemented and we can remove the rating component from bury list

Reply:
But it's breaking change.

Reply:
Should we notice it in the "Breaking changes" part of PR or just revert changes to make the rating facet hidden as it was before?

Reply:
It's still actual.